modules/services/disciplina.nix

```diff
@@ -0,0 +1,81 @@
+{ config, lib, pkgs, ... }: with lib;
+
+let
+  cfg = config.services.disciplina;
+
+  attrsToFlags = set:
+    let
+      render = name: value:
+        "--" + name + (optionalString (isString value) (" " + value));
```

Comment:
just ignore it if it's not a string, whatever?

Reply:
```
gen-key = true; → --gen-key
```